plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/jdbc/TestIcebergJdbcCatalogConnectorSmokeTest.java

```diff
@@ -46,6 +55,8 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     protected QueryRunner createQueryRunner()
             throws Exception
     {
+        warehouseLocation = Files.createTempDirectory("test_iceberg_jdbc_catalog_smoke_test").toFile();
```

Comment:
The value of `warehouseLocation` is different from `iceberg.jdbc-catalog.default-warehouse-dir` set in `IcebergQueryRunner`. Probably, we should set the property explicitly in the following `setIcebergProperties` and remove `if` condition from `IcebergQueryRunner`. Otherwise, the current variable name is misleading.

Reply:
I have added a separate commit for this.

Reply:
Thanks. I believe `TestIcebergJdbcConnectorTest` will fail.

Reply:
Thanks, @ebyhr for the review. Yes, `TestIcebergJdbcConnectorTest` was failing without the change. Fixed it now.